tests/test_ykman.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+try:
+    from ykman.device import list_all_devices
+    YKMAN_AVAILABLE = True
+except ImportError:
+    YKMAN_AVAILABLE = False
+
+@pytest.mark.skipif(not YKMAN_AVAILABLE, reason="ykman module is not available")
```

Comment:
I think this `skipif` will work, but `test_repository_tool` should be updater and `taf.yubikey` should probably be surrounded by a similar try-except.

Reply:
I have edited the code accordingly. Hope this seems reasonable.